specification/eventhub/resource-manager/Microsoft.EventHub/stable/2024-01-01/Clusters.json

```diff
@@ -0,0 +1,581 @@
+            }
+          }
+        },
+        "x-ms-long-running-operation": true
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}/namespaces": {
+      "get": {
+        "tags": [
+          "EventHubs Clusters Namespace"
+        ],
+        "operationId": "Clusters_ListNamespaces",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "description": "List all Event Hubs Namespace IDs in an Event Hubs Dedicated Cluster.",
+        "x-ms-examples": {
+          "ListNamespacesInCluster": {
+            "$ref": "./examples/Clusters/ListNamespacesInClusterGet.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "List of Namespace IDs in Cluster successfully delivered.",
+            "schema": {
+              "$ref": "#/definitions/EHNamespaceIdListResult"
+            }
+          },
+          "default": {
+            "description": "Event Hubs Cluster error response detailing why this operation failed.",
+            "schema": {
+              "$ref": "../../../common/v2/definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.EventHub/clusters/{clusterName}/triggerUpgrade": {
+      "post": {
+        "tags": [
+          "EventHubs Clusters Upgrade"
+        ],
+        "operationId": "Clusters_TriggerUpgradePost",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ClusterNameParameter"
+          },
+          {
+            "$ref": "../../../common/v1/definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "description": "Trigger pending cluster upgrade by skipping customer-set upgrade preferences",
+        "x-ms-examples": {
+          "TriggerUpgradeInCluster": {
+            "$ref": "./examples/Clusters/TriggerUpgradeInCluster.json"
+          }
+        },
+        "responses": {
+          "204": {
+            "description": "Successfully bypassed customer-set upgrade preferences on pending cluster upgrade."
```

Comment:
>ly bypassed customer-set upgrade preferences on pending cluster upgrade." [](http://example.com/codeflow?start=38&length=73)

It seems more to the point to me to say 

"Successfully started applying pending cluster updates?"

Since thats whats actually happening (as a result of bypassing preferences)?

If I understood it all right.